### PR TITLE
PlusCal variant of EWD998.

### PR DESCRIPTION
**Pull request by lemmy**
@muenchnerkindl Vanilla PlusCal variant of EWD998.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ Here is a list of specs included in this repository, with links to the relevant
 | [Misra Reachability Algorithm](specifications/MisraReachability)                                    | Leslie Lamport                                      |          |      ✔      |    ✔    |     ✔     |          |
 | [Byzantizing Paxos by Refinement](specifications/byzpaxos)                                          | Leslie Lamport                                      |          |      ✔      |    ✔    |     ✔     |          |
 | [EWD840: Termination Detection in a Ring](specifications/ewd840)                                    | Stephan Merz                                        |          |      ✔      |         |     ✔     |          |
-| [EWD998: Termination Detection in a Ring with Asynchronous Message Delivery](specifications/ewd998) | Stephan Merz, Markus Kuppe                          |          |      ✔      |         |     ✔     |          |
+| [EWD998: Termination Detection in a Ring with Asynchronous Message Delivery](specifications/ewd998) | Stephan Merz, Markus Kuppe                          |          |      ✔      |   (✔)   |     ✔     |          |
```
Comment (ahelwer) [inline README.md:41]: I don't know how the parentheses thing started, must have missed it - could you remove them and remove the parentheses from the other one that has them?

Comment (lemmy) [reply README.md:41]: I don't see a better way to indicate that a primary TLA+ spec also comes with an alternative Pluscal spec. Without parents, readers might think it is only a Pluscal spec.

Comment (ahelwer) [reply README.md:41]: Perhaps then also add a note about the parentheses meaning above the table.

Comment (lemmy) [reply README.md:41]: https://github.com/tlaplus/Examples/pull/125/commits/9bed7d54bc73e0f1cce978a34491f8825ea0cb59#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R12

File: manifest.json [hunk 2]
```diff
@@ -4330,6 +4330,28 @@
...
+          "path": "specifications/ewd998/EWD998PCal.tla",
+          "communityDependencies": [
+            "BagsExt"
+          ],
+          "tlaLanguageVersion": 2,
+          "features": ["pluscal"],
+          "models": [
+            {
+              "path": "specifications/ewd998/EWD998PCal.cfg",
+              "runtime": "00:01:00",
```
Comment (ahelwer) [inline manifest.json:4343]: If the runtime is one minute this spec doesn't count as a small spec; those are for specs that execute in less than 10 seconds.

**Pull request merged**